### PR TITLE
Apex test for Account OwnerId sync

### PR DESCRIPTION
**Pull request by eclemmer**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+
+/**
+ * @description T-______: __________
+ */
+@isTest
+private with sharing class LeadConversionAccountNaming_FTST {
```
Comment (xinnanlisf) [inline unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls:33]: i noticed this empty class was added to this branch. Was it on purpose?

Comment (salvadorsfdc) [reply unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls]: @eclemmer I was wondering the same Xinnan is asking here.

File: unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (salvadorsfdc) [inline unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls-meta.xml]: It seems this file is also extra

File: unpackaged/functional_tests/lead_conversion/package.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (salvadorsfdc) [inline unpackaged/functional_tests/lead_conversion/package.xml]: It seems this file is also extra

File: force-app/main/default/classes/ACCT_IndividualAccounts_TEST.cls [hunk 4]
```diff
@@ -2696,4 +2696,4 @@ private class ACCT_IndividualAccounts_TEST {
 
         System.assertEquals('test 0 Administrative Account', accAfterUpdate.Name);
     }
-}
+}
```
Comment (salvadorsfdc) [inline force-app/main/default/classes/ACCT_IndividualAccounts_TEST.cls:2699]: I'm not sure we should be touching this file as well?

**Review by salvadorsfdc — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/ACCT_IndividualAccounts_TEST.cls (hunk 4 above)
Comment (salvadorsfdc) [reply force-app/main/default/classes/ACCT_IndividualAccounts_TEST.cls:2699]: nevermind, it seems a weird github thing

File: unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls (hunk 1 above)
Comment (eclemmer) [reply unpackaged/functional_tests/lead_conversion/classes/LeadConversionAccountNaming_FTST.cls]: Removed, thanks!

**Review by salvadorsfdc — APPROVED** [review]
changes look good, approving.

**Pull request merged**